File: VB/SpreadWinDemo/print/printsetting.vb
Public Class printsetting

    Public Sub New()

        ' この呼び出しはデザイナーで必要です。
        InitializeComponent()

        ' InitializeComponent() 呼び出しの後で初期化を追加します。

        ' シートの設定
        InitSheet(FpSpread1.Sheets(0))
    End Sub

    Private Sub InitSheet(sheet As FarPoint.Win.Spread.SheetView)
        ' データの作成
        Dim dt As New DataTable()
        Dim cols As Integer = 20
        Dim rows As Integer = 65
        For i As Integer = 0 To cols - 1
            dt.Columns.Add(String.Format("列{0}", i + 1), GetType(String))
        Next
        For i As Integer = 0 To rows - 1
            Dim dr As DataRow = dt.NewRow()
            For j As Integer = 0 To cols - 1
                dr(j) = String.Format("R{0}C{1}", i, j)
            Next
            dt.Rows.Add(dr)
        Next
        dt.AcceptChanges()

        ' データ連結
        sheet.DataAutoCellTypes = False
        sheet.DataAutoSizeColumns = False
        sheet.DataSource = dt

        ' セルノートの設定
        sheet.Cells(1, 1).Note = "セルノートの表示"
        sheet.Cells(1, 1).NoteIndicatorSize = New Size(5, 5)
    End Sub

    Private Sub button1_Click(sender As Object, e As System.EventArgs) Handles button1.Click
        ' マウスカーソルの一時変更
        Me.Cursor = System.Windows.Forms.Cursors.WaitCursor

        ' ページ設定ダイアログの表示
        Dim psDialog As New PrintSettingForm(FpSpread1)
        psDialog.ShowDialog()

        ' マウスカーソルの一時変更の解除
        Me.Cursor = System.Windows.Forms.Cursors.Default
    End Sub
End Class

File: CS/SpreadWinDemoCS/print/printsetting.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SpreadWinDemo.print
{
    public partial class printsetting : SpreadWinDemo.DemoBase
    {
        public printsetting()
        {
            InitializeComponent();

            // シートの設定
            InitSheet(fpSpread1.Sheets[0]);

            button1.Click += new EventHandler(button1_Click);
        }

        private void InitSheet(FarPoint.Win.Spread.SheetView sheet)
        {
            // データの作成
            DataTable dt = new DataTable();
            int cols = 20;
            int rows = 65;
            for (var i = 0; i < cols; i++)
            {
                dt.Columns.Add(string.Format("列{0}", i + 1), typeof(String));
            }
            for (var i = 0; i < rows; i++)
            {
                DataRow dr = dt.NewRow();
                for (var j = 0; j < cols; j++)
                {
                    dr[j] = string.Format("R{0}C{1}", i, j);
                }
                dt.Rows.Add(dr);
            }
            dt.AcceptChanges();
            
            // データ連結
            sheet.DataAutoCellTypes = false;
            sheet.DataAutoSizeColumns = false;
            sheet.DataSource = dt;

            // セルノートの設定
            sheet.Cells[1, 1].Note = "セルノートの表示";
            sheet.Cells[1, 1].NoteIndicatorSize = new Size(5, 5);
        }

        void button1_Click(object sender, EventArgs e)
        {
            // マウスカーソルの一時変更
            this.Cursor = System.Windows.Forms.Cursors.WaitCursor;

            // ページ設定ダイアログの表示
            PrintSettingForm psDialog = new PrintSettingForm(fpSpread1);
            psDialog.ShowDialog();

            // マウスカーソルの一時変更の解除
            this.Cursor = System.Windows.Forms.Cursors.Default;
        }
    }
}
